cpp/src/text/bpe/load_merge_pairs.cu

```diff
@@ -107,8 +107,9 @@ std::unique_ptr<bpe_merge_pairs::bpe_merge_pairs_impl> create_bpe_merge_pairs_im
   rmm::cuda_stream_view stream,
   rmm::device_async_resource_ref mr)
 {
+  auto space = std::string(" ");  // workaround to ARM issue
```

Comment:
We should open a cuDF issue documenting where we are with this investigation and link to all the relevant PRs, then include that issue number here.

Also `const`?
```suggestion
  auto const space = std::string(" ");  // workaround to ARM issue
```